packages/twenty-server/src/engine/metadata-modules/flat-view-group/utils/from-create-view-group-input-to-flat-view-group-to-create.util.ts

```diff
@@ -8,23 +8,25 @@ export const fromCreateViewGroupInputToFlatViewGroupToCreate = ({
   createViewGroupInput: rawCreateViewGroupInput,
   workspaceId,
   workspaceCustomApplicationId,
+  mainGroupByFieldMetadataId,
 }: {
   createViewGroupInput: CreateViewGroupInput;
   workspaceId: string;
   workspaceCustomApplicationId: string;
+  mainGroupByFieldMetadataId: string;
 }): FlatViewGroup => {
-  const { fieldMetadataId, viewId, ...createViewGroupInput } =
+  const { viewId, ...createViewGroupInput } =
     trimAndRemoveDuplicatedWhitespacesFromObjectStringProperties(
       rawCreateViewGroupInput,
-      ['fieldMetadataId', 'fieldValue', 'id', 'viewId'],
+      ['fieldValue', 'id', 'viewId'],
     );
 
   const createdAt = new Date();
   const viewGroupId = createViewGroupInput.id ?? v4();
 
   return {
     id: viewGroupId,
-    fieldMetadataId,
+    fieldMetadataId: mainGroupByFieldMetadataId, // Mandatory because non-nullable until we completely remove it
```

Comment:
Understood, we will wait for the migration to be run
We could still already start omitting the fieldMetadataId from the `FlatViewGroup` tab to be sure we're not still using it anywhere